### PR TITLE
Update hatecrime and privacy footer es i8n

### PR DESCRIPTION
**Pull request by Jkrzy**
[Link to ZenHub issue.](https://app.zenhub.com/workspaces/doj-crt-intake-scrum-board-5d03bf56c1c8a35d482eca0f/issues/18f/crt-portal/542)

## What does this change?
Adding in latest spanish translations for
hatecrime question and privacy policy
footer.

## Screenshots (for front-end PR):
Screen Shot 2020-05-26 at 9 52 15 PM

Screen Shot 2020-05-26 at 9 50 49 PM

### Author

+ [x] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [x] Check for [accessibility](/docs/a11y_plan.md).
+ [x] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

**Comment by Jkrzy** [comment]
🤔 Not sure why pa11y is failing on the link color contrast

**Comment by LindsayYoung** [comment]
The error seems to be on the landing page, where I am seeing a contrast issue - but it isn't from this PR- I see it on dev too. 

Screen Shot 2020-05-26 at 7 20 21 PM

Now that the pages are longer, I think we need to adjust the wait for commands for something at the end of the pages. I think that is why the a11y tests have been inconsistent lately.

**Comment by Jkrzy** [comment]
Ahhhh thanks @LindsayYoung! A fix for that is included in #497

**Comment by Jkrzy** [comment]
@LindsayYoung Pushed a small tweak to the `circleci` config and upload pa11y artifacts, results here are :interesting:.

Not sure why it appears to be only loading _some_ of the CSS
pa11y

File: .circleci/config.yml [hunk 1]
```diff
@@ -60,7 +60,7 @@ jobs:
             curl --retry-delay 5 --retry 10 --retry-connrefused http://127.0.0.1:8000
             npm run test:a11y
       - store_artifacts:
-          path: /pa11y-screenshots
+          path: ./pa11y-screenshots
```
Comment (LindsayYoung) [inline .circleci/config.yml:63]: <3 Thank you I had been meeting to look in to this for sooo long!!!

**Comment by Jkrzy** [comment]
Ooooo, it's because `runserver` doesn't serve static files unless `DEBUG = True`. So the pa11y tests in CI have maybe been running without styles

**Comment by Jkrzy** [comment]
😌 

Rebased against `develop` to resolve conflicts and made updates necessary for pa11y suite to execute as expected and pass.

Includes: 
1. Adding of SR-only labels for accessible autocomplete
2. Modifying django settings/circleCI config so pa11y tests execute with css loaded
3. Removed a single pa11y test configuration in `crt-view` which felt a bit duplicative and was failing unexpectedly/inexplicable.

This can be replaced by #506 if desired

**Comment by Jkrzy** [comment]
Replaced by #507

**Pull request closed without merging**